### ISSUE TITLE
Aniwave Failed To Fetch Videos

### ISSUE THREAD

**Issue by YouSummoner** [aniyomiorg/aniyomi-extensions#2520]
### Source information

Aniwave.to

### Source language

English

### Steps to reproduce

1. Try to play a video on Aniwave extension 

### Expected behavior

The video should load.

### Actual behavior

It gives an error saying "Failed To Fetch Videos" 

![Screenshot_20231117-195032_Aniyomi](https://github.com/aniyomiorg/aniyomi-extensions/assets/107224555/851ef2f8-84dc-41b8-8532-43c4f7de892c)

### Aniyomi version

Preview r6691

### Android version

Android 13

### Other details

This is only happening on Airtel India network, but just so you know, Aniwave.to isn't blocked on Airtel, the website works just fine and plays videos normally, it's something related to the extension itself.

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3.10](https://github.com/aniyomiorg/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/aniyomiorg/aniyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by LittleFreak** [comment]
Which Aniwave version do you have installed?
Do you run any adblocker like pihole?

**Comment by YouSummoner** [comment]
> Which Aniwave version do you have installed? Do you run any adblocker like pihole?
Aniwave version 13.58
And No, i don't have any sort of adblock installed
